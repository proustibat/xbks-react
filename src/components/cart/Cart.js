import React from 'react';
import CartList from './CartList';
import CartTotal from './CartTotal';

export class Cart extends React.Component {
    render () {
        return (
            <div className="cart">
                <CartList />
                <CartTotal />
            </div>
        );
    }
}

export default Cart;
