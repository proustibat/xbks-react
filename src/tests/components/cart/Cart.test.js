import React from 'react';
import { shallow } from 'enzyme';
import Cart from "../../../components/cart/Cart";

test( 'Should render Cart component correctly', () => {
    const wrapper = shallow( <Cart /> );
    expect( wrapper ).toMatchSnapshot();
} );
