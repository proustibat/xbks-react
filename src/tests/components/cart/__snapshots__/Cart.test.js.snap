// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Cart component correctly 1`] = `
<div
  className="cart content-container"
>
  <Connect(CartList) />
  <Connect(CartTotal) />
</div>
`;
